src/public/log/schema.rs

```diff
@@ -1135,11 +1141,20 @@ mod test {
 
         assert!(ocsf.is_ok());
         let ocsf_log = ocsf.unwrap();
-        assert_eq!(ocsf_log.severity_id, SeverityId::Medium);
-        assert_eq!(ocsf_log.status.unwrap(), "Failure".to_string());
+        assert_eq!(ocsf_log.severity_id, SeverityId::Informational);
```

Comment:
Yea, but I tried it out and it was able to generate a single error. You can modify this to generate a second error. You don't need to take in an int to generate `N` number of errors, you can just have a case for 0, 1, and 2 errors. Try this for the 1 case. Your current CR has the 0 case. 

Hint:

You can change `generate_response` to take in `Option<bool>` then you can do:

```
match input (maybe call this multiple_errors?) {
   Ok(true) => // 2 error case
   Ok(false) => // 1 error case 
   None => // no error case
}
```

Reply:
Why would this be better than taking in an int?

Reply:
I think it would be easier. Generating a policy that can create any N number of errors would require you to manipulate the policy to generate any number of errors. Not sure what that policy would look like. Maybe you can copy-paste whats inside the when clause to generate extra errors?

You can go that way if you want, but the tests only require a 0, 1, and 2 case. Its not really idiomatic to take an int if the output only has 3 states. Your choice on how you go about it.